### ISSUE TITLE
Maybe remove "Short-circuiting is cleaner than conditionals"?

### ISSUE THREAD

**Issue by adamshone** [ryanmcdermott/clean-code-javascript#71]
Because:

a) You already have "Use default arguments instead of short circuiting", which is a better approach (if supported), and

b) I have seen bugs introduced by writing a short circuit when the function is supposed to accept falsy values:

```
function double(x) {
   var value = x || -1;
   return value * 2;
}

double(0);  // returns -2, should return 0
```

**Comment by JefferyHus** [comment]
Because 0 is considered as false all time, that's why you will keep getting -2

**Comment by ryanmcdermott** [comment]
Hey there Adam! I think this should now be fixed after #34
